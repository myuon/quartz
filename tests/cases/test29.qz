fun main() {
    let a = 100;
    println(a.to_string());

    let a = 200;
    println(a.to_string());

    for a in 0..2 {
        for b in 0..3 {
            println("{}:{}", a.to_string(), b.to_string());
        }
    }

    println(a.to_string());

    if true {
        let a = 300;
        println(a.to_string());
    }

    println(a.to_string());

    println("ok");
}

