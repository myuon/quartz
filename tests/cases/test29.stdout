100
200
0:0
0:1
0:2
1:0
1:1
1:2
200
300
200
ok
